3p/nameframe.max.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
Also, [use two-space indents throughout the file](https://google.github.io/styleguide/htmlcssguide.xml?showone=Indentation#Indentation).